packages/components/src/panel/README.md

```diff
@@ -99,64 +104,68 @@ The `PanelBody` creates a collapsible container that can be toggled open or clos
 
 ##### Props
 
-###### title
+###### title: `string`
 
-Title of the `PanelBody`. This shows even when it is closed.
+Title text. It shows even when it is closed.
```

Comment:
Thanks for the review @chad1008! @ciampo initially suggested this change in his review, and also suggested:

>Since we're in the PanelBody props, we can also refer to PanelBody as "the component"

So would it make sense to change it to the following?

`Title of the component. It shows even when it is closed.`

Reply:
That could work! Or maybe something like

`Title text. It shows even when the component is closed.`? Either could work!